game/card.go

```diff
@@ -24,7 +26,7 @@ type Card struct {
 	Name                 CardName
 	PhyrexianCastingCost *CastingCost
 	Powermenace          bool // only blockable by >= power (like Skarrgan Pitskulk)
-
+	Subtype              MTGSubtype
```

Comment:
Let's not name some things "MTGX". Because basically everything here is a MTG something, so why would some of them get named "MTGX" and others just named "X". Also let's use the same terms mtg does. It looks like things are called "card types", "subtypes", and "supertypes". http://mtg.wikia.com/wiki/Card_Types So how about just CardType, Subtype, and Supertype